commands/tests/accountabilityPins.test.js

```diff
@@ -58,13 +70,17 @@ describe('Accountability when outside correct channel', () => {
 	});
 
 	test('"Unpin All" does not call handler when not in accountability channel', async () => {
-		jest.mock('../../config.json', () => {
```

Comment:
Just out of curiosity, why was this changed? New linting rules?

Reply:
I added `{ virtual: true }` as a third parameter to this function call to allow the test to mock `config.json` even if the file isn't there. I moved the first parameter down to line up with the other two